scale.yml

```diff
@@ -1,5 +1,6 @@
 ---
 - hosts: localhost
+  gather_facts: False
```

Comment:
I'm not sure that `ansible_version` is defined `gather_facts` is set to false...
Maybe `become: false` is enough

Reply:
`gather_facts` is not necessary since `ansible_version` is hardcoded into the local ansible installation. To verify that I've added a `debug` task before the first one:

```
- hosts: localhost
  gather_facts: False
  tasks:
    - debug:
        msg: "Ansible version is {{ ansible_version.string }}"
    - name: "Check ansible version !=2.7.0"
      assert:
        msg: "Ansible V2.7.0 can't be used until: https://github.com/ansible/ansible/issues/46600 is fixed"
        that:
          - ansible_version.string is version("2.7.0", "!=")
          - ansible_version.string is version("2.5.0", ">=")
      tags:
        - check
  vars:
    ansible_connection: local
```

and the output is:

```
PLAY [localhost] **************************************************************************************************************************************************************

TASK [debug] ******************************************************************************************************************************************************************
Thursday 08 November 2018  12:14:40 +0000 (0:00:00.184)       0:00:00.184 *****
ok: [localhost] => {
    "msg": "Ansible version is 2.6.7"
}

TASK [Check ansible version !=2.7.0] ******************************************************************************************************************************************
Thursday 08 November 2018  12:14:40 +0000 (0:00:00.050)       0:00:00.235 *****
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}
```